libnmstate/ifaces/ovs.py

```diff
@@ -255,7 +255,7 @@ def _validate_ovs_mtu_mac_confliction(self):
 def is_ovs_running():
     try:
         subprocess.run(
-            ("systemctl", "status", "openvswitch"),
+            ("ovsdb-client", "list-dbs"),
```

Comment:
This will add a dependency on containers, can we use the python OVS client to do same operation ? similar tu the stuff under plugin/ovs...

Reply:
New approach uploaded.

Instead of doing pre-check, I removed the `is_ovs_running()` and trust nmstate-NM plugin raise the proper exception when OVSDB malfunctioned.

Introducing new dependency to nmstate or nmstate-NM plugin is not ideal in this case, the `ovsdb` plugin is not install by default hence it can hold extra dependency.